### ISSUE TITLE
Missing variants

### ISSUE THREAD

**Issue by kpjonsson** [BRCAChallenge/brca-exchange#981]
Hi,

I'm querying the API as instructed in the tutorial notebook. 

```
httpClient = client.HttpClient("https://brcaexchange.org/backend/data/ga4gh/v0.6.0a7/")

query = httpClient.search_variants(reference_name = 'chr17', variant_set_id = "brca-hg37", start = 41228504, end = 41228506) 
```

I expect this query to find this variant https://brcaexchange.org/variant/180754, but it does not. Any clue what's wrong?

**Comment by kpjonsson** [comment]
Hi @zfisch. Any update or comment on this?

**Comment by zfisch** [comment]
Hi @kpjonsson, apologies for the delayed response. I haven't had a chance yet to delve into this but it's on my radar.

In the meantime, what are you hoping to accomplish by retrieving this data? Maybe I can point you to a workaround?

**Comment by kpjonsson** [comment]
@zfisch: I'm querying a set of BRCA1 and BRCA2 variants, mainly to retrieve the pathogenicity annotation from ENIGMA. As far as I'm aware that annotation isn't readily available anywhere else. Is that true?

**Comment by zfisch** [comment]
Gotcha -- we're working through some issues at the moment but hopefully the following will get you what you need in the meantime.

You can always download the latest data from brcaexchange.org at https://brcaexchange.org/backend/downloads/releases/current_release.tar.gz. After unzipping, navigate to output/release/built_with_change_types.tsv and you can find all of the data on our site in tsv format. There's also a README in the root folder with information about the different files in output/.

Another option is to download data from the variants page. Navigate to https://brcaexchange.org/variants then click the download button. Make sure you're in Detail View to access the download button (you can change views by clicking the Detail View / Summary View icon in the header).

<img width="1209" alt="brca_exchange" src="https://user-images.githubusercontent.com/4503616/52669105-24f7c580-2eca-11e9-8820-0408102f3f10.png">

Does this get you what you need or do you have any further questions?

**Comment by kpjonsson** [comment]
Yep, this is great. The variant is in there and this table is easy enough to query. Thank you.

**Comment by kpjonsson** [comment]
I really think you should keep this issue open unless you think that the issue is resolved. But that is your choice.

**Comment by zfisch** [comment]
@kpjonsson fair, reopening for future consideration
